src/api/goods.ts

```diff
@@ -3,17 +3,59 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+export const getAll = async (): Promise<Good[]> => {
   return fetch(API_URL)
     .then(response => response.json());
-}
+};
+
+export const get5First = async () => {
```

Comment:
can't try it, but it seems like it doesn't work as expected

Reply:
https://www.loom.com/share/1e6e99128c81414990ae9858205af089

Reply:
https://www.loom.com/share/1e6e99128c81414990ae9858205af089 
it works. but  It works slowly on the video